### ISSUE TITLE
Question: Can the "production" contracts (0.4.0) be used with latest Raiden 

### ISSUE THREAD

**Issue by offerm** [raiden-network/raiden#3752]
Should I be able to use the "production" contracts (0.4.0) when using the latest raiden (master) on  a private chain?

If I face problems - should I report?

**Comment by LefterisJP** [comment]
Yes they should be, when putting `--environment-type production`. At the moment there are problems with latest master though which https://github.com/raiden-network/raiden/pull/3732 should be solving.

**Comment by offerm** [comment]
So, I deployed contract version 0.4.0 and started raiden with `--environment-type production`

All looks as expected. 

Yet, I don't see the token I created and registered:

```
xud@xud-test-1:~$ curl -i http://localhost:5002/api/v1/tokens
HTTP/1.1 200 OK
mimetype: application/json
Content-Type: application/json
Content-Length: 2
Date: Fri, 05 Apr 2019 08:59:07 GMT
[]xud@xud-test-1:~$ 
```

Any idea?

**Comment by offerm** [comment]
@LefterisJP I managed to solve it (I hope). Let me explain.

I'm using 2 machines. first machine is running geth, second machine is running raiden. 
I created the contracts on the first machine. 

To solve the problem I had to copy the file `/home/xud/raiden-contracts/raiden_contracts/data_0.4.0/deployment_private_net.json`

to `~/.virtualenvs/raiden/lib/python3.7/site-packages/raiden_contracts/data_0.4.0/` on the second machine.

a) why is this file needed if the contract addresses are given to raiden on the command line
b) deploying contracts with multiple raidens is somehow problematic
